tables.md

```diff
@@ -0,0 +1,204 @@
+# Tables
+
+When writing your JSON-LD file to accompany your csv, you can write it in a way to include multiple csvs.
+
+Below is an extract of two csv that will be described in the JSON-LD.
+
+The first one is called football.csv.
+
+| period_type | period_code | variable_name     | observation | measure | unit       |
+| ----------- | ----------- | ----------------- | ----------- | ------- | ---------- |
+| Year        | 2023        | Manchester United | 6           | billion | U.S dollar |
+| Year        | 2023        | Liverpool         | 5.3         | billion | U.S dollar |
+| Year        | 2023        | Manchester City   | 5           | billion | U.S dollar |
+| Year        | 2023        | Chelsea           | 3.1         | billion | U.S dollar |
+| Year        | 2023        | Tottenham Hotspur | 2.8         | billion | U.S dollar |
+
+The second one is american_football.csv.
+
+| period_type | period_code | variable_name        | observation | measure | unit       |
+| ----------- | ----------- | -------------------- | ----------- | ------- | ---------- |
+| Year        | 2023        | Dallas Cowboys       | 9           | billion | U.S dollar |
+| Year        | 2023        | New England Patriots | 7           | billion | U.S dollar |
+| Year        | 2023        | Los Angeles Rams     | 6.9         | billion | U.S dollar |
+| Year        | 2023        | New York Giants      | 6.8         | billion | U.S dollar |
+| Year        | 2023        | Chicago Bears        | 6.3         | billion | U.S dollar |
+
+There are additional descriptors for human readability that you can add to your column sections. These include:
+
+- `suppressOutput`  If true, suppresses any output that would be generated when converting cells in this column. The value of this property becomes the suppress output annotation for the described column. The default is false.(from w.3/org-tabular-metadata)
+-  `propertyuri`, `valueurl`, `aboutUrl`  This metadata creates a relationship model between data in each column by different combinations of aboutUrl, propertyUrl, and valueUrl on existing columns, and defining new virtual columns to supply additional information. (from w.3/org-tabular-metadata)
+- `label` this can be used for additional context.
+
+
+## Table Breakdown
+
+This section will help you how to write your table section of your JSON-LD file. But also, how to describe each column from your csv file. 
+
+The first section will show you what needs to be done for each csv you are using.
+
+The second section will go through a step by step process of writing and describing the columns in your csv.
+
+## First Section
```

Comment:
What would be a better section header here? Is one eve necessary?

Reply:
I have renamed it. Is this acceptable.